src/style.css

```diff
@@ -1,3 +1,66 @@
+:root {
+  --primary-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+header {
```

Comment:
do not use tag selector - add class and use `.header` selector
fix all similar cases